utils/releaser/src/main.rs

```diff
@@ -14,13 +17,24 @@ enum Releaser {
     ReleaseApple,
     ReleaseAndroid,
     ReleaseWindows,
+    ReleasePublicSite,
 }
 
 fn main() {
+    root();
     match Releaser::from_args() {
         Releaser::DeployServer => server::deploy_server(),
         Releaser::ReleaseApple => apple::release_apple(&Github::env(), &AppStore::env()),
         Releaser::ReleaseAndroid => android::release_android(&Github::env(), &PlayStore::env()),
         Releaser::ReleaseWindows => windows::release(&Github::env()),
+        Releaser::ReleasePublicSite => public_site::release(),
     }
 }
+
+pub fn root() -> PathBuf {
```

Comment:
Kind of feels like a util function, but either way, I'd throw a comment next to the vague `root();` line in main.

Reply:
yep you're right

Reply:
I will fix & merge